src/Npgsql/Util/CodeAnalysis/AllowNullAttribute.cs

```diff
@@ -0,0 +1,11 @@
+﻿#if NET461 || NETSTANDARD2_0 || NETSTANDARD2_1
+#pragma warning disable 1591
+// ReSharper disable once CheckNamespace
+namespace System.Diagnostics.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, Inherited = false)]
+    public sealed class AllowNullAttribute : Attribute
```

Comment:
OK, so it's confirmed that on older TFMs the compiler respects these attributes? Also, are they really not included in .NET Standard 2.1?

Reply:
Unfortunately, not included.